### PR TITLE
deploy deb packages to old and new web host

### PR DESCRIPTION
**Pull request by evgeni**
note: should not be merged before 2.2 RC1 is out.

### CODE REVIEW

File: puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/release.groovy [hunk 1]
```diff
@@ -23,6 +23,7 @@ void push_rpms_katello(version) {
 void push_debs_direct(os, repo) {
     sshagent(['freight']) {
         sh "ssh freight@web02.theforeman.org deploy ${os} ${repo}"
+        sh "ssh freight@web01.osuosl.theforeman.org deploy ${os} ${repo}"
```
Comment (ekohl) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/release.groovy:26]: Does it make sense to do this in parallel?

**Comment by evgeni** [comment]
Yeah. But I'd prefer one breakage at a time ;)

File: puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/debian/stage_packages.sh [hunk 2]
```diff
@@ -13,13 +13,15 @@ DEB_PATH=./debian/${os}/build-${project}
...
-TARGET_PATH="${USER}@${HOST}.theforeman.org:rsync_cache/${os}/${COMPONENT}/"
+for HOST in HOSTS; do
+  # The path is important, as freight_rsync (which is run on the web node for incoming
+  # transfers) will parse the path to figure out the repo to send debs to.
+  TARGET_PATH="${USER}@${HOST}.theforeman.org:rsync_cache/${os}/${COMPONENT}/"
 
-if ls $DEB_PATH/*deb >/dev/null 2>&1; then
-  /usr/bin/rsync -avPx $DEB_PATH/*deb $TARGET_PATH
-fi
+  if ls $DEB_PATH/*deb >/dev/null 2>&1; then
```
Comment (ekohl) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/debian/stage_packages.sh:24]: Perhaps this if should be outside of the for loop? Same applies to other scripts.

Comment (evgeni) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/debian/stage_packages.sh:24]: technically yes, but I didn't want to also change the logic flow of the script, and the loop needs to be outside of `TARGET_PATH` as that contains `HOST`.

Comment (ekohl) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/debian/stage_packages.sh:24]: We should really rewrite these to Jenkins pipelines. But I'm ok with merging it as is.

Comment (evgeni) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/debian/stage_packages.sh:24]: Absolutely!

**Pull request merged**